Documentation/UserTsconfig/Setup.rst

```diff
@@ -28,6 +28,30 @@ setup.default.[someProperty]
     existing users is by :guilabel:`Reset Backend User Preferences` in the
     :guilabel:`Admin tools > Maintenance` section of the install tool.
 
+    .. code-block:: typoscript
+       :caption: EXT:site_package/Configuration/admin.tsconfig
```

Comment:
```suggestion
       :caption: EXT:site_package/Configuration/user.tsconfig
```

Since v13 `Configuration/user.tsconfig` is read automatically. So we should use that.

Reply:
Ah, didn't know that. Thanks ;-)